validator-api/src/monitor/mod.rs

```diff
@@ -137,9 +136,9 @@ impl Monitor {
     }
 
     async fn test_run(&mut self) {
-        info!(target: "Monitor", "Starting test run no. {}", self.nonce);
+        info!("Starting test run no. {}", self.nonce);
```

Comment:
This is just a question, what is the reason for removing the `target` part of the macro?

Reply:
Great that you raised this, `rocket` overrode the logging macros, so I made them compatible to make it compile, forgot to put them back